plugins/woocommerce-admin/client/marketing/data-multichannel/types.ts

```diff
@@ -50,7 +50,31 @@ export type RecommendedChannelsState = {
 	error?: ApiFetchError;
 };
 
+export type Campaign = {
+	id: string;
+	channel: string;
+	title: string;
+	manage_url: string;
+	cost: {
+		value: string;
+		currency: string;
+	};
+};
+
+export type CampaignsPage = {
+	data?: Array< Campaign >;
+	error?: ApiFetchError;
+};
+
+export type CampaignsState = {
+	error?: ApiFetchError;
```

Comment:
❓  Whats the difference between the error in `CampaignsState` vs an error in  `CampaignsPage `??

Reply:
Done in [`4b87f0a` (#36735)](https://github.com/woocommerce/woocommerce/pull/36735/commits/4b87f0a4ed13d00bd8054687bdab823ba10a6a03).